src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -382,15 +382,14 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
 
         [Fact]
         [OuterLoop]
-        [ActiveIssue(12778)]
         public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
         {
             // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
             const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
             
             using (var client = new HttpClient())
             {
-                client.Timeout = TimeSpan.FromSeconds(60);
+                client.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
@ianhays, feel free to go ahead with your timeout change as you and David discussed; I can revise the whole test soon (unless you beat me to it).